backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -12,7 +12,43 @@ const {
   app: { notmEndpointUrl: dpsUrl },
 } = require('../../config')
 
-module.exports = ({ prisonApi, logError }) => async (req, res) => {
+const fetchStaffName = (context, staffId, prisonApi) =>
+  prisonApi.getStaffDetails(context, staffId).then(staff => formatName(staff.firstName, staff.lastName))
+
+const fetchReasonDescription = (context, assignmentReasonCode, caseNotesApi) => {
+  if (assignmentReasonCode) {
+    return caseNotesApi
```

Comment:
Be really interested to see what the errors look like when they fail on the mapping part when making an API call.